spec/support/specialist_publisher_assertion_helpers.rb

```diff
@@ -29,5 +35,17 @@ def expect_unpublished
     expect(find(".alert").text).to match(/^Unpublished/)
   end
 
+  def reload_while_failing
+    attempts = 1
+    begin
+      yield
+    rescue Capybara::ElementNotFound, RSpec::Expectations::ExpectationNotMetError => e
+      raise e if attempts >= Capybara.default_max_wait_time
```

Comment:
Ah yeah, oops. The attempts was tied into the wait value before I refactored some of this which pulled it. I'll need to have a re-think. Thanks.